docs/user/ml/index.asciidoc

```diff
@@ -98,3 +98,44 @@ image::user/ml/images/outliers.png[{oldetection-cap} results in {kib}]
 
 For more information about the {dfanalytics} feature, see
 {ml-docs}/ml-dfanalytics.html[{ml-cap} {dfanalytics}].
+
+[[xpack-ml-aiops]]
+== AIOps
+
+AIOps is a part of {ml-app} in {kib} which provides features that use advanced 
+statistical methods to help you interpret your data and its behavior.
+
+[discrete]
+[[explain-log-rate-spikes]]
+=== Explain log rate spikes
+
+preview::[]
+
+Explain log rate spikes is a feature that uses advanced statistical methods to 
+identify reasons for increases in log rates. It makes it easy to find unusual 
+spikes by using the investigative workflow view. Examine the histogram chart of 
+the log rates for a given {data-source}, and find the reason behind a particular 
+change possibly in millions of log events across multiple fields and values.
+
+You can find explain log rate spikes under **{ml-app}** > **AIOps** where you 
+can select the {data-source} or saved search that you want to analyze.
+
+[role="screenshot"]
+image::user/ml/images/ml-explain-log-rate-before.png[Log event histogram chart]
+
+Select a spike in the log event histogram chart to start the analysis. It 
+identifies statistically significant field-value combinations that contribute to 
+the spike and displays them in a table. The table also shows an indicator of the 
+level of impact and a sparkline showing the shape of the impact in the chart. 
+Hovering over a row displays the impact on the histogram chart in more detail. 
+
+Sliders in the chart show the baseline values and the deviation in the analyzed 
+data. You can move the sliders to redefine both the baseline and the deviation 
+and rerun the analysis with the modified values.
```

Comment:
Suggest to use "brushes" instead of "sliders". Might be an edge case, but I think "sliders" are more general/isolated UI elements to select something on a range, whereas "brushes" are the infovis related technical term. But maybe we could even use something like "interactive annotations" or similar? Instead of "baseline values" suggest to use "baseline time range".